tests/Collection/CrudSpecFunctionalTest.php

```diff
@@ -39,8 +43,16 @@ public function testSpecification(array $initialData, array $test, $minServerVer
         $expectedData = isset($test['outcome']['collection']['data']) ? $test['outcome']['collection']['data'] : null;
         $this->initializeData($initialData, $expectedData);
 
-        $result = $this->executeOperation($test['operation']);
-        $this->executeOutcome($test['operation'], $test['outcome'], $result);
+        $result = null;
+        $exception = null;
+
+        try {
+            $result = $this->executeOperation($test['operation']);
+        } catch (RuntimeException $e) {
+            $exception = $e;
```

Comment:
This assignment isn't strictly necessary. The variable will still exist outside the ``catch`` scope. You can just do ``catch (RuntimeException $exception)``.

Reply:
Noted. Just wanted to err on the side of clarity here, which is why I also initialized `$exception` above as `null`.